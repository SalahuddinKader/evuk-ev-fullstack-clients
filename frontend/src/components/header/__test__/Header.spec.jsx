import React from 'react';
import { render } from '@testing-library/react';
import Header from '..';

describe('Header Component', () => {
  it('matches the snapshot', () => {
    const { baseElement } = render(<Header />);
    expect(baseElement).toMatchSnapshot();
  });
});
