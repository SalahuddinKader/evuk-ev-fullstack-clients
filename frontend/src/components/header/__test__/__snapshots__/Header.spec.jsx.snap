// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component matches the snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiBox-root css-17btvqz"
    >
      <h4
        class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
        data-testid="header"
      >
        Clients List App
      </h4>
    </div>
  </div>
</body>
`;
